src/test/kotlin/org/rust/lang/core/stubs/RsStubAccessTest.kt

```diff
@@ -45,6 +48,15 @@ class RsStubAccessTest : RsTestBase() {
         processStubsWithoutAstAccess<RsElement> { it.reference }
     }
 
+    fun `test inner outer attributes does not need ast`() {
+        processStubsWithoutAstAccess<RsElement> { element ->
+            if (element is RsInnerAttributeOwner && element !is RsFile)
+                element.innerAttrList
```

Comment:
Let's use braces for blocks even with one line action except blocks with single `return` expr and `if (condition) expr1 else expr2` expressions

Reply:
Done, i also removed 2 unnecessary implementations of `val outerAttrList: List<RsOuterAttr>`